src/test/kotlin/com/zopa/ktor/opentracing/KtorOpenTracingTest.kt

```diff
@@ -415,26 +416,6 @@ class KtorOpenTracingTest {
         } }
     }
 
-    @Test
-    fun `UuidFromPath returns unchanged path and no uuid if no UUID in path`() {
-        val path = "/evidence"
-
-        val pathUuid = path.UuidFromPath()
-
-        assertThat(pathUuid.path).isEqualTo(path)
-        assertThat(pathUuid.uuid).isEqualTo(null)
-    }
-
-    @Test
-    fun `UuidFromPath returns path with UUID and uuid if UUID in path`() {
-        val path = "/evidence/AB7AD59A-A0FF-4EB1-90CF-BC6D5C24095F"
-
-        val pathUuid = path.UuidFromPath()
-
-        assertThat(pathUuid.path).isEqualTo("/evidence/<UUID>")
-        assertThat(pathUuid.uuid).isEqualTo("AB7AD59A-A0FF-4EB1-90CF-BC6D5C24095F")
-    }
```

Comment:
can we keep these tests we're still using it in the client feature

Reply:
Whoops yes good point